src/Fixer/ClassNotation/ClassAttributesSeparationFixer.php

```diff
@@ -39,6 +39,11 @@
  */
 final class ClassAttributesSeparationFixer extends AbstractFixer implements ConfigurableFixerInterface, WhitespacesAwareFixerInterface
 {
+    /**
+     * @internal
+     */
+    public const SPACING_HYBRID = 'hybrid';
```

Comment:
`hybrid` is not very explicit about what it does: I suggest `one_if_phpdoc`.

Reply:
I had the same remark, but could not find a good name, kudos @julienfalque for finding it.